apple/internal/apple_support_toolchain.bzl

```diff
@@ -152,6 +156,13 @@ A `File` referencing a dynamic library used to redirect stdout and stderr when n
             executable = True,
             doc = """
 A `File` referencing a tool that copies and lipos Swift stdlibs required for the target to run.
+""",
+        ),
+        "symbols_tool": attr.label(
+            cfg = "host",
```

Comment:
This should be `exec`, though all other darwin tools here use the `host` configuration, so I just keep it the same.